components/automate-ui/src/app/entities/destinations/destination.requests.ts

```diff
@@ -139,12 +139,24 @@ export class DestinationRequests {
       return this.testDestinationWithSecretId(destination);
     }
   }
-
   public testDestinationWithUsernamePassword(url: string,
     username: string, password: string): Observable<Object> {
-    return this.http.post(encodeURI(
-      this.joinToDataFeedUrl(['destinations', 'test'])),
-      { url, 'username_password': {username, password} });
+        return this.http.post(encodeURI(
+          this.joinToDataFeedUrl(['destinations', 'test'])),
+          { url, 'username_password': {username, password} });
+  }
+
+  public testDestinationWithUsernamePasswordWithHeaders(url: string,
```

Comment:
Please use headers to send username password as basic auth header. The backend does not support receiving of username_password and headers at the same time.